web_tree_many2one_clickable/static/src/js/web_tree_many2one_clickable.js

```diff
@@ -0,0 +1,105 @@
+//-*- coding: utf-8 -*-
+//############################################################################
+//
+//   OpenERP, Open Source Management Solution
+//   This module copyright
+//     (C) 2013 Therp BV (<http://therp.nl>).
+//     (c) 2015 Serv. Tecnol. Avanzados (http://www.serviciosbaeza.com)
+//              Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+//
+//   This program is free software: you can redistribute it and/or modify
+//   it under the terms of the GNU Affero General Public License as
+//   published by the Free Software Foundation, either version 3 of the
+//   License, or (at your option) any later version.
+//
+//   This program is distributed in the hope that it will be useful,
+//   but WITHOUT ANY WARRANTY; without even the implied warranty of
+//   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//   GNU Affero General Public License for more details.
+//
+//   You should have received a copy of the GNU Affero General Public License
+//   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+//############################################################################
+
+openerp.web_tree_many2one_clickable = function(instance, local)
+{
+    instance.web.list.Column.include({
+        /*
+        Load config parameter at init and store it in an accessible variable.
+        */
+        init: function(id, tag, attrs) {
+            this._super(id, tag, attrs);
+            if (this.type == 'many2one') {
+                this.get_options();
+            }
+        },
+
+        get_options: function() {
+            if (_.isUndefined(this.ir_options_loaded)) {
+                var self = this; // Needed for binding the instance
+                this.ir_options_loaded = $.Deferred();
+                this.use_many2one_clickable = false;
+                (new instance.web.Model("ir.config_parameter"))
+                    .query(["value"])
+                    .filter([['key', '=', 'web_tree_many2one_clickable.default']])
+                    .first()
+                    .then(function(param) {
+                        self.use_many2one_clickable = (param.value == 'true');
+                        self.ir_options_loaded.resolve();
+                    });
+                return this.ir_options_loaded;
+            }
+            return $.when();
+        },
+
+        _format: function (row_data, options)
+        {
+            if (this.use_many2one_clickable) {
+                return _.str.sprintf('<a class="oe_form_uri" data-many2one-clickable-model="%s" data-many2one-clickable-id="%s">%s</a>',
```

Comment:
how about a refactoring so that this and instance.web_tree_many2one_clickable.Many2OneClickable._format call the same function on `instance.web.list.Column`?